src/sys/socket/mod.rs

```diff
@@ -826,6 +842,14 @@ pub enum ControlMessage<'a> {
               target_os = "android",
               target_os = "ios",))]
     Ipv6PacketInfo(&'a libc::in6_pktinfo),
+
+    // SO_RXQ_OVFL indicates that an unsigned 32 bit value
```

Comment:
Did you mean to make this a doc comment instead of a regular comment?

Reply:
Changed to `///`.